src/app_service/service/token/service.nim

```diff
@@ -401,7 +401,7 @@ QtObject:
   proc getTokenBySymbolByContractAddr(self: Service, contractAddr: string): TokenBySymbolItem =
```

Comment:
hmm I thought the token service only managed ETH/ERC20, how is it related to ENS?